src/utils/logging.js

```diff
@@ -1,28 +1,48 @@
 import { DEBUG } from '../config'
 
 /**
- * Logs something if the env is not PROD, otherwise does nothing
+ * A logging level reference can be passed as first parameter to the log function. The levels
+ * correspond to the nativr console methods.
+ */
+export const LOGLEVEL = {
+    ERROR: 'ERROR',
+    INFO: 'INFO',
+    DEBUG: 'DEBUG',
+}
+
+/**
+ * Logs something if the env is not PROD, otherwise does nothing.
  *
- * @param {String} level
+ * @param {String} [level]
  * @param {Boolean | String | Number | Object} message
  */
 const log = function (level, ...message) {
     /* eslint-disable no-console */
     if (DEBUG) {
         switch (level) {
-            case 'error':
+            case LOGLEVEL.ERROR:
                 console.error(...message)
                 break
-            case 'info':
+            case LOGLEVEL.INFO:
                 console.info(...message)
                 break
-            case 'debug':
+            case LOGLEVEL.DEBUG:
                 console.debug(...message)
                 break
             default:
-                console.log(...message)
+                console.log(level, ...message)
         }
     }
     /* eslint-enable  no-console */
 }
+
+/* eslint-disable no-console */
+/** Same as console.error but doesn't log in production. */
+log.error = (...message) => console.error(...message)
+/** Same as console.info but doesn't log in production. */
+log.info = (...message) => console.info(...message)
+/** Same as console.debug but doesn't log in production. */
+log.debug = (...message) => console.debug(...message)
```

Comment:
Maybe we could call back to `log()` function here, and modify how we call `console.xxx` there. This way we only use console.log and other in the main `log()` function

Reply:
Yeah. The shorthands were a bit ugly. I updated them to use our `log` function so they can respect the `DEBUG` constant.